### DISCUSSION TITLE
extend the highlight displaying the differences over all cells of the line

### DISCUSSION THREAD

**Discussion by nbehrnd** [borgbase/vorta#1274] [Ideas]
For tab «archives», button «diff» eventually yields an additional window «dialog» to see if changes between two backups are due to addition, remove, or modification of files and directories.  For this, a tap on the first cell of two backups selects these and highlights their complete line of record (date, duration, ... name) with a distinct light blue background.

In the case of «dialog» to display the differences of two backups, this however does not extend over all cells of the line in question.  To illustrate:

![2022-04-24_vorta_highlight_deltas](https://user-images.githubusercontent.com/19709084/164973959-6cf9fbee-360a-4541-9f7e-833802d4e970.png)

For the example above, it would be nice if the highlight equally would include both a) the cell about why vorta/borg identify the file/directory as different.  And b) if the different background would include the the third cell, about the original volume the file/directory.

The observations refer to vorta 0.8.3 (package 0.8.3-1) as provided by the repositories of Linux Debian 12/bookworm (branch testing).

**Comment by real-yfprojects** [comment]
Do I understand you correctly that you would appreciate the highlight spanning the entire row?

**Comment by nbehrnd** [comment]
Affirmative, I would appreciate the highlight spanning the entire
row.

**Comment by nbehrnd** [comment]
Please consider the conversation as closed because vorta version 0.8.10 does provide a highlight extending all across the corresponding line:

![2023-02-14_vorta_highlight_selection](https://user-images.githubusercontent.com/19709084/218678432-cb8299d9-003f-4a6f-a358-955e70bf6fe3.png)

If hovering over the entry, the complete path to the file in question is displayed, too (for the clarity of the illustration, this is not shown here).

**Reply by real-yfprojects** [reply]
Yeah, I implemented that.
